### PR TITLE
PSP-2300 update organization

### PR DESCRIPTION
**Pull request by asanchezr**
image

### CODE REVIEW

File: backend/api/Areas/Organizations/Mapping/Organization/AddressMap.cs [hunk 1]
```diff
@@ -24,8 +25,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.CountryId, src => src.Address.CountryId)
                 .Map(dest => dest.Postal, src => src.Address.PostalCode)
                 .Map(dest => dest.CountryOther, src => src.Address.OtherCountry)
-                .Map(dest => dest.AddressTypeId, src => src.AddressUsageTypeCodeNavigation)
-                .Inherits<Entity.IBaseAppEntity, Api.Models.BaseAppModel>();
```
Comment (FuriousLlama) [inline backend/api/Areas/Organizations/Mapping/Organization/AddressMap.cs:28]: should this line do the mapping to rowversion instead?

File: backend/api/Areas/Persons/Mapping/Person/PersonMap.cs [hunk 2]
```diff
@@ -42,6 +41,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.PimsPersonAddresses, src => src.Addresses)
                 .Map(dest => dest.PimsContactMethods, src => src.ContactMethods)
                 .Map(dest => dest.PimsPersonOrganizations, src => src.Organization != null ? new[] { src } : null)
+                .Inherits<Api.Models.BaseAppModel, Entity.IBaseAppEntity>()
```
Comment (FuriousLlama) [inline backend/api/Areas/Persons/Mapping/Person/PersonMap.cs:44]: nice!

File: backend/api/Areas/Organizations/Mapping/Organization/AddressMap.cs (hunk 1 above)
Comment (asanchezr) [reply backend/api/Areas/Organizations/Mapping/Organization/AddressMap.cs:28]: Sorry I don't understand the question. Could you be more specific please? 
Is it that the mapping is wrong?

Comment (asanchezr) [reply backend/api/Areas/Organizations/Mapping/Organization/AddressMap.cs:28]: Ah I understand the question now. If you look closely the RowVersion comes from the sub-entity -> `src.Address.ConcurrencyControlNumber`

This is due to flattening the model (which we won't do moving forward but this is old code). The generic mapping wants the RowVersion from the main entity (PimsOrgAddress)

**Pull request merged**